src/index.html

```diff
@@ -1,9 +1,241 @@
+    <meta content="yes" name="mobile-web-app-capable">
+    <link href="assets/favicon/manifest.json" rel="manifest">
+    <link href="assets/favicon/apple-touch-icon.png" rel="apple-touch-icon" sizes="180x180">
+    <link href="assets/favicon/favicon-32x32.png" rel="icon" sizes="32x32" type="image/png">
+    <link href="assets/favicon/favicon-16x16.png" rel="icon" sizes="16x16" type="image/png">
+    <link href="assets/favicon/manifest.json" rel="manifest">
+    <link color="#017dc3" href="assets/favicon/safari-pinned-tab.svg" rel="mask-icon">
+    <meta content="#ffffff" name="theme-color">
+    <link href="assets/images/favicon.ico?" rel="shortcut icon" type="image/x-icon">
+    <title>
+        Coastal Resilience – Mapping portal
+    </title>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,600,700" rel="stylesheet">
+    <link href="assets/css/main.css" rel="stylesheet">
+</head>
+
+<body>
+    <div class="app-container">
+        <div class="sidebar" data-js-all-worksites="">
+            <div class="sidebar--actions hide-desktop">
+                <a href="#" id="sidebar-toggle">
+                        List all worksites
+                    </a>
+            </div>
+            <div class="sidebar--header">
+                <div class="logo">
+                    <span class="sr-only">
+                            Coastal Resilience
+                        </span>
+                    <img src="assets/images/cr-logo-white.png" alt="Coastal Resilience">
+                </div>
+                <h1 class="site-title">
+                    Mapping portal
+                </h1>
+                <button class="btn btn-primary" data-toggle="modal" data-target="#about-modal" id="modal-toggle-btn">
+                        Learn more
+                    </button>
+            </div>
+            <div class="sidebar--inner">
+                <div id="region-list-view"></div>
+                <script type="text/template" id="regionListTemplate">
+                    <div class="location--list">
+                        <p>
+                            At consectetur, reprehenderit ad distinctio sapiente obcaecati facilis dolor possimus officia incidunt consequatur in sed dignissimos neque.
+                        </p>
+                        <% _.each(regionList, function(region) { %>
+                            <div class="location" demo-js-parent-location id="<%= region.title %>">
+                                <div class="location--title" id="<%= region.title %>">
+                                    <%= region.title %>
+                                </div>
+                                <img src="<%= region.image %>" alt="<%= region.title %>">
+                            </div>
+                            <% }) %>
+                    </div>
+                </script>
+                <div id="region-details-view"></div>
+                <script type="text/template" id="regionDetailsTemplate">
```

Comment:
Minor issue, but in most projects (PWD, the GeositeFramework), templates are all grouped together at the bottom of `index.html`, rather than placed inline near where they will be rendered. I think the idea is that it's easier to find them if they are all in one place than scattered throughout the DOM.